enclone_args/src/read_json.rs

```diff
@@ -262,28 +262,14 @@ fn parse_vector_entry_from_json(
             print_some_annotations(refdata, &ann, &mut log, false);
             print!("\n{}", strme(&log));
         }
-        let mut log = Vec::<u8>::new();
```

Comment:
Is this actually used at all now?  If it starts empty and isn't mutable, it's presumably unused.